### DISCUSSION TITLE
Can I use the repo to do color quantization?

### DISCUSSION THREAD

**Discussion by zydjohnHotmail** [emgucv/emgucv#714] [Ideas]
Hello:
I want to know if I can use this repo to get color quantization for an image.
There are a lot of such repo for Python, as indicated by this web page:
https://github.com/topics/color-quantization
But there no such repo in C#.
Please advise,
Thanks,

**Comment by emgucv** [comment]
Once you pick the color quantization method you want in python, it shouldn't be too hard to covert it into c#. The  majority of the python functions are implemented in c# as well. If there is something missing, feel free to let us know and we will work on  it.

**Comment by juacamgo93** [comment]
> Hello: I want to know if I can use this repo to get color quantization for an image. There are a lot of such repo for Python, as indicated by this web page: https://github.com/topics/color-quantization But there no such repo in C#. Please advise, Thanks,

Hi, you can do color quantization using kmeans algorithm, that it is included in EmguCV.

There is a lot of information on the web about color quantization using kmeans clustering, check any of them and apply the same knowledge to EmguCV and C#.

Functions using EmguCV will not change between Python's OpenCV and this wrapper, you only need to take care on the C# translation from Python.
